doc/events.md

```diff
@@ -2,13 +2,13 @@
 
 ## Introduction
 Events are messages about actions that have occurred over LXD. Using the API endpoint `/1.0/events` directly or via
-`lxc monitor` will connect to a WebSocket through which logs and lifecycle messages will be streamed.
+`lxc monitor` will connect to a WebSocket through which logs and life-cycle messages will be streamed.
 
 ## Event types
 LXD Currently supports three event types.
 - **Logging**: Shows all logging messages regardless of the server logging level.
 - **Operation**: Shows all ongoing operations from creation to completion (including updates to their state and progress metadata).
-- **Lifecycle**: Shows an audit trail for specific actions occurring over LXD.
+- **Life cycle**: Shows an audit trail for specific actions occurring over LXD.
```

Comment:
I think those should be using `logging`, `operation` and `lifecycle` instead of the bold text they're using today.
That would line them up with the actual event type as present in the API or in `lxc monitor`